### PR TITLE
Handle click banner song

### PR DESCRIPTION
**Pull request by VuManhHung97**
Handle when click banner will play music

https://edu-redmine.sun-asterisk.vn/issues/13119

### CODE REVIEW

**Comment by VuManhHung97** [comment]
ready

File: app/src/main/java/com/sun_asterisk/music_65/utils/OnItemViewpagerClickListener.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.sun_asterisk.music_65.utils;
+
+import com.sun_asterisk.music_65.data.model.Song;
+import java.util.List;
+
+public interface OnItemViewpagerClickListener {
+    void onItemBannerClickListener(List<Song> songs, int position);
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/music_65/utils/OnItemViewpagerClickListener.java]:
```suggestion
    void onItemRecyclerViewClickListener(List<Song> songs, int position);
```

**Comment by VuManhHung97** [comment]
ready

File: app/src/main/java/com/sun_asterisk/music_65/screen/allsong/adapter/BannerPagerAdapter.java [hunk 2]
```diff
@@ -12,28 +12,37 @@
...
     private TextView mTextViewBannerHome;
+    private OnItemViewpagerClickListener mListener;
+    private int mPosition;
 
     public BannerPagerAdapter(Context context) {
         mContext = context;
         mSongs = new ArrayList<>();
     }
 
+    public void setOnItemBannerViewClickListener(OnItemViewpagerClickListener listener) {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/music_65/screen/allsong/adapter/BannerPagerAdapter.java]:
```suggestion
    public void setOnItemClickListener(OnItemViewpagerClickListener listener) {
```

File: app/src/main/java/com/sun_asterisk/music_65/utils/OnItemViewpagerClickListener.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.sun_asterisk.music_65.utils;
+
+import com.sun_asterisk.music_65.data.model.Song;
+import java.util.List;
+
+public interface OnItemViewpagerClickListener {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/music_65/utils/OnItemViewpagerClickListener.java]:
```suggestion
public interface OnItemClickListener {
```

**Comment by VuManhHung97** [comment]
ready

File: app/src/main/java/com/sun_asterisk/music_65/utils/OnItemClickListener.java [hunk 4]
```diff
@@ -0,0 +1,8 @@
+package com.sun_asterisk.music_65.utils;
+
+import com.sun_asterisk.music_65.data.model.Song;
+import java.util.List;
+
+public interface OnItemClickListener {
+    void onItemRecyclerViewClickListener(List<Song> songs, int position);
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/music_65/utils/OnItemClickListener.java]:
```suggestion
    void onItemRecyclerViewClick(List<Song> songs, int position);
```

**Comment by VuManhHung97** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**